x-pack/plugins/security_solution/public/detections/mitre/mitre_tactics_techniques.ts

```diff
@@ -10547,7 +10826,7 @@ export const subtechniquesOptions: MitreSubtechniquesOptions[] = [
  *
  * Is built alongside and sampled from the data in the file so to always be valid with the most up to date MITRE ATT&CK data
  */
-export const getMockThreatData = () => ({
+export const mockThreatData = {
```

Comment:
I think this is the thing causing all the ci errors since this is used elsewhere in the app for tests but aside from that looks all good!

Reply:
Thank you for the 👀's @dplumlee! 🙂 

Looks like this was manually modified here: https://github.com/elastic/kibana/pull/105162/files#diff-eb9f3696e4dc5a15306c231aaceff87f7d26c23ed861d6ef096b5ef8a5c8cc14R10146

Will update the generator script and put a `THIS IS A GENERATED FILE` warning at the top as well.